cmd/openshift/operator/kodata/tekton-addon/addons/01-clustertriggerbindings/gitlab.yaml

```diff
@@ -34,6 +36,8 @@ spec:
     value: $(body.commits[0].message)
   - name: git-repo-url
     value: $(body.repository.git_http_url)
+  - name: git-repo-ssh-url
+    value: $(body.project.git_ssh_url)
```

Comment:
this should be body.repository.git_ssh_url according to https://docs.gitlab.com/ee/user/project/integrations/webhook_events.html#push-events